subscriptionform/templates/sports/team_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
```

Comment:
Ik heb geprobeerd om in de view de `template_name` te overschrijven naar een generieke `confirm_delete.html` maar dat werkt niet. Suggesties?